testdata/ksym.c

```diff
@@ -0,0 +1,39 @@
+#include "common.h"
+
+char __license[] __section("license") = "Dual MIT/GPL";
+
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__uint(max_entries, 2);
+	__type(key, uint32_t);
+	__type(value, uint64_t);
+} array_map __section(".maps");
+
+static long (*bpf_map_update_elem)(void *map, const void *key, const void *value, uint64_t flags) = (void *)2;
```

Comment:
This is already in `common.h` as `map_update_elem`. I'll send a quick patch to remove this from the other testdata files that do the same so the copypasta stops.